src/controllers/movie.js

```diff
@@ -0,0 +1,94 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+// Get all movies
+const getMovies = async (req, res) => {
```

Comment:
This function is named well and is self-descriptive so the comment above it is what could be considered a "dead comment"